tests/core/test_blob_utils.py

```diff
@@ -201,12 +205,20 @@ def test_chunk_root_calculation():
     assert calc_chunk_root(body) == calc_merkle_root(chunks)
 
 
-@pytest.mark.parametrize("blobs,unpadded_body", BLOB_SERIALIZATION_TEST_DATA)
+@pytest.mark.parametrize(
+    "blobs,unpadded_body",
+    BLOB_SERIALIZATION_TEST_DATA,
+    ids=BLOB_SERIALIZATION_TEST_IDS,
```

Comment:
I didn't know about the `ids` arg to `parametrize()`, nice!  It'd be good to have a comment here explaining why we want to specify the ids ourselves, though